src/node.h

```diff
@@ -303,9 +303,16 @@ NODE_EXTERN void RunAtExit(Environment* env);
 NODE_EXTERN struct uv_loop_s* GetCurrentEventLoop(v8::Isolate* isolate);
 
 /* Converts a unixtime to V8 Date */
-#define NODE_UNIXTIME_V8(t) v8::Date::New(v8::Isolate::GetCurrent(),          \
-    1000 * static_cast<double>(t))
-#define NODE_V8_UNIXTIME(v) (static_cast<double>((v)->NumberValue())/1000.0);
+NODE_DEPRECATED("Use v8::Date::New() directly",
+                inline v8::Local<v8::Value> NODE_UNIXTIME_V8(double time) {
```

Comment:
This is not equivalent.
In this case I could imagine a user define time related struct, with no implicit cast to `double`, e.g.
```c++
struct Timestamp {
    double _val;
    explicit operator double() { return _val; };
};
```

Reply:
While that’s true, I think it’s highly unlikely that such code which uses this method exists.